src/main/java/org/eclipse/yasson/internal/serializer/MapSerializer.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021, 2022 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, 2025 Oracle and/or its affiliates. All rights reserved.
```

Comment:
We should not modify the copyright dates in the file. If you rebase on the main branch then the check will not fail.